setup.py

```diff
@@ -120,6 +120,7 @@ def gen_packages_items():
 
 
 if __name__ == '__main__':
+    lmdeploy_package_data = ['lmdeploy/bin/*']
```

Comment:
All of executables in `lmdeploy/bin/` are packed?

Reply:
only llama_gemm is installed to `lmdeploy/bin/`

Reply:
Can we specify `lmdeploy/bin/llama_gemm`?